### ISSUE TITLE
Empty body-parameter names fail with swagger-ui 3.*

### ISSUE THREAD

**Issue by ikitommi** [metosin/reitit#399]
Swagger `:body` parameter names default to empty string, which fails with swagger-ui 3.*, making the body-parameters uneditable. Reitit should default to non-empty string, like `body`.

**Comment by ikitommi** [comment]
fixed in 0.5.0

**Comment by taraktikos** [comment]
It is still reproducible in 0.5.2
```
["/objects" {:get  {:summary "Find all objects"
                                  :handler h/find-objects}
                           :post {:summary    "Create object"
                                  :parameters {:body {:title string?}}
                                  :responses  {200 {:body {:title string?}}}
                                  :handler    h/create-object}}]
```
in swagger.json

```
"/api/v1/objects": {
      "get": {
        "responses": {
          "default": {
            "description": ""
          }
        },
        "produces": [
          "application/json",
          "application/transit+msgpack",
          "application/transit+json",
          "application/edn"
        ],
        "consumes": [
          "application/json",
          "application/transit+msgpack",
          "application/transit+json",
          "application/edn"
        ],
        "parameters": [],
        "summary": "Find all objects",
        "tags": [
          "api / v1"
        ]
      },
      "post": {
        "responses": {
          "200": {
            "schema": {
              "type": "object",
              "properties": {
                "title": {
                  "type": "string"
                }
              },
              "required": [
                "title"
              ]
            },
            "description": ""
          }
        },
        "produces": [
          "application/json",
          "application/transit+msgpack",
          "application/transit+json",
          "application/edn"
        ],
        "consumes": [
          "application/json",
          "application/transit+msgpack",
          "application/transit+json",
          "application/edn"
        ],
        "parameters": [
          {
            "in": "body",
            "name": "",
            "description": "",
            "required": true,
            "schema": {
              "type": "object",
              "properties": {
                "title": {
                  "type": "string"
                }
              },
              "required": [
                "title"
              ]
            }
          }
        ],
        "summary": "Create object",
        "tags": [
          "api / v1"
        ]
      }
    },
```
